### DISCUSSION TITLE
<title> Lost Access to 2-Factor Authentication

### DISCUSSION THREAD

**Discussion by prammos** [community/community#127719] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi all,

I have previously linked my personal github account (xxxx@gmail.com) with my professional account (xxxx@company.com).
My employer required 2FA, which I had set up on my work laptop. I have now left this company, so I have all lost methods of 2FA.
Can I still access my personal github repos at xxxx@gmail.com?

I can't seem to find the answer [here](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). 

I have tried contacting "[support](https://support.github.com/contact/account?contact[type]=account&flow=2fa)", but all I am getting is a very unhelpful Virtual Assistant.

PS. I made a different account to be able to post this.

Thank you.

**Comment by davevad93** [comment]
Hi @prammos ,

Try with opening a ticket [here](https://support.github.com/contact/account), select `Two-factor authentication recovery` and explain your issue.

**Reply by prammos** [reply]
Thank you!

**Reply by davevad93** [reply]
You're welcome!
